pyne/alara.py

```diff
@@ -843,3 +852,64 @@ def _get_subvoxel_array(mesh, cell_mats):
 
     return subvoxel_array
 
+_TO_SEC = {
+    's': 1,
+    'second': 1,
+    'm': 60,
+    'minute': 60,
+    'h': 60 * 60,
+    'hour': 60 * 60,
+    'd': 60 * 60 * 24,
+    'day': 60 * 60 * 24,
+    'w': 60 * 60 * 24 * 7,
+    'week': 60 * 60 * 24 * 7,
+    'y': 60 * 60 * 24 * 365,
+    'year': 60 * 60 * 24 * 365,
+    'c': 60 * 60 * 24 * 365 * 100,
+    'century': 60 * 60 * 24 * 365 * 100,
```

Comment:
Years should be 365.25 days

Reply:
This change has been made in the update.